src/tiled/tiled.pro

```diff
@@ -260,7 +260,8 @@ SOURCES += aboutdialog.cpp \
     wangfiller.cpp \
     wangtemplateview.cpp \
     wangtemplatemodel.cpp \
-    zoomable.cpp
+    zoomable.cpp \
+    InvertYCoordinateHelper.cpp
```

Comment:
Please keep the list sorted alphabetically when adding new files (I know the IDE doesn't do it for qmake files, so it needs to be done manually).